tests/AutoReview/ProjectCodeTest.php

```diff
@@ -121,7 +121,9 @@ function (\ReflectionClass $interface): array {
             'configure', // due to AbstractFixer::configure
             'getConfigurationDefinition', // due to AbstractFixer::getConfigurationDefinition
             'getDefaultConfiguration', // due to AbstractFixer::getDefaultConfiguration
+            'registerCustomRuleSets', // due to waiting to be added in 4.0 to ConfigInterface
             'setWhitespacesConfig', // due to AbstractFixer::setWhitespacesConfig
+            'registerCustomRuleSets', // due to waiting to being added in 4.0 to ConfigInterface
```

Comment:
I'm confused why it's added twice

Reply:
This has to be a mistake. Likely a mishap in a rebase/merge.